README.md

```diff
@@ -17,25 +17,28 @@ https://discord.gg/5JmwPJf
 - OpenGL (Version >= 3.3)
 
 #### Building
+Building of this project is a simple endeavour, requiring only a few commands.
 
-- Either clone https://github.com/GentenStudios/quartz-engine or fork and then clone your fork. You will have to either clone with the
-``--recursive`` option, or you can run: ``git submodule update --init`` after cloning.
+Initially, a [clone](https://help.github.com/en/articles/cloning-a-repository) of the repository will be required, however, if you plan on making edits first, make sure you [fork](https://help.github.com/en/articles/fork-a-repo) the repo beforehand, and clone that.
 
-- Run `cmake -H. -Bbuild` inside the root directory
+An example command for cloning the repository is:
 
-- Then either build with `cmake --build build` or follow the instructions for your compiler
+<dl>
+git clone https://github.com/GentenStudios/quartz-engine.git
+</dl>
 
-  ##### Visual Studio
+Once cloned, enter the directory, by doing `cd quartz-engine``, and complete the following commands:
 
-  - Right click on `quartz-sandbox` and click `Set as StartUp project`
-  - Click `Local Windows Debugger` to run
+<dl>
+cmake -H. -BBuild
+cmake --build Build
+</dl>
```

Comment:
I still oppose teaching users how to CD, if they don't know how they need a tutorial on command line basics and if they do know how then this is unnecessary fluff.